components/vf-lede/vf-lede.scss

```diff
@@ -10,7 +10,7 @@
 @import 'vf-lede.variables.scss';
 
 .vf-lede {
-  @include set-type(body--xl, $custom-margin-bottom: 0);
+  @include set-type(body--xl, $custom-margin-bottom: 2rem);
```

Comment:
I the reason this was set to `0` was for use on the group pages. Does this need to be a mixin, or a new `.vf-intro .vf-lede` added?

![image](https://user-images.githubusercontent.com/928100/60801286-0f37b500-a177-11e9-8047-d9757b159380.png)

Reply:
hmmm, overall vertical spacing still needs a look at - perhaps we should move this to the `vf-intro` component and have it add the margin there. 

I'm not sure if it needs to mixin.

Once we get more pages built out - we can start seeing how the vertical spacing of all the things fits (or doesn't).

Reply:
Moving the tweak to `vf-intro` sounds like a cleaner mid-term approach.

And +1 to revisiting spacing all around.